src/org/labkey/test/tests/DomainDesignerTest.java

```diff
@@ -0,0 +1,265 @@
+package org.labkey.test.tests;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.domain.DomainResponse;
+import org.labkey.remoteapi.domain.GetDomainCommand;
+import org.labkey.remoteapi.query.SaveRowsResponse;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.SortDirection;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.DailyA;
+import org.labkey.test.components.DomainDesignerPage;
+import org.labkey.test.components.DomainFieldRow;
+import org.labkey.test.components.DomainFormPanel;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.ListHelper;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.TestDataGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@Category({DailyA.class})
```

Comment:
any specific one?  I picked it arbitrarily.

Reply:
Actually. I think putting it in a standard suite is probably just fine. I was thinking this would be a temporary test but we probably want to make it more permanent if we're going to start leveraging APIs to do things more widely.
That said, put it in DailyB or DailyC. DailyA is, by far, the longest running of the three.

Reply:
DailyB it shall be